src/admin/components/website/modalParts/WebsiteFeedbackEmailOverride.tsx

```diff
@@ -90,44 +106,63 @@ class WebsiteFeedbackEmailOverride extends React.Component <any, any> {
 		});
 	}
 
-	render () {
-		const feedbackEmailOverride = this.props.feedbackEmailOverride.map((email, index) => {
-			return (
+	buildList () {
+		return this.props.feedbackEmailOverride.map((email, index) => (
 				<li key={index + '-email-override'} className="website-feedback-email-override-list-item">
 					{email}
 					<i className="fa fa-times" onClick={this.onDelete.bind(this, index)}/>
-				</li>);
-		});
+				</li>));
+	}
+
+	render () {
+		const {value, touched} = this.state;
+		const {settings: {feedback: feedbackOverrideStatus = false}} = this.props;
 		return (
-			<div className="medium-12 columns">
+			<div className="medium-12 columns website-feedback-email-override-container overrides-container">
```

Comment:
Do you really need such long classnames?

Reply:
For this moment yes, my opinion for styles it would be great to refactor them to work similar with some kind of class-name, not setting for each one, but before it is done